.github/workflows/guix.yml

```diff
@@ -0,0 +1,92 @@
+name: ci/gh-actions/guix
+
+on:
+  push:
+    paths:
+      - 'contrib/depends/**'
+      - 'contrib/guix/**'
+      - '!contrib/**.md'
+      - '.github/workflows/guix.yml'
+  pull_request:
+    paths:
+      - 'contrib/depends/**'
+      - 'contrib/guix/**'
+      - '!contrib/**.md'
+      - '.github/workflows/guix.yml'
+
+jobs:
+  cache-sources:
+    runs-on: ubuntu-24.04
```

Comment:
Should we switch to debian (12)?

Reply:
Oh, that might fix the AppArmor issue. Good suggestion, will try.